vignettes/geologic-map.Rmd

```diff
@@ -23,10 +23,10 @@ load_all()
 <div style="text-align: justify">
 
 # Introduction
-`rmacrostrat` is an R package that allows users to easily retrieve geologic data from the [Macrostrat](https://macrostrat.org) database and facilitate analyses within the R environment. This vignette (or tutorial if you prefer) is provided to guide you through the installation process and some of the functionality available within `rmacrostrat`. To do so, we will focus on making a geographic map of a specific geologic formation, a commonly required (and very important) figure within a variety of papers across the geosciences. 
+`rmacrostrat` is an R package that allows users to easily retrieve geologic data from the [Macrostrat](https://macrostrat.org) database and facilitates analyses of these data within the R environment. This vignette (or tutorial, if you prefer) is provided to guide you through the installation process and some of the functionality available within `rmacrostrat`. Specifically, we will focus on making a geographic map of a specific geologic formation, a commonly required (and very important) figure for a variety of purposes across the geosciences. 
```

Comment:
Maybe this is a stupid question but is there a formal system for when to use 'geologic' vs. 'geological'? I don't think this is super important but maybe should be consistent throughout the documentation.

Reply:
I had a check of this when doing some writing for the paper; I always thought that geologic was American english, whereas geological was more UK, but apparently there isn't a real clear distinction between the two and they can be used somewhat interchangably. But yes, happy to make it so we stick to one or the other.